tests/animation/newgenanim.test.ts

```diff
@@ -1977,7 +1980,7 @@ describe('NewGen Anim', () => {
             // Start Motion -> Empty
             updater.goto(FIRST_TIME_EMPTY_REST_TIME + MOTION_EXIT_CONDITION + 0.15);
             expectAnimationGraphEvalStatus(graphEval, [
-                { current: { clip: clipMotion!.clip, weight: 1.0 - 0.15 / MOTION_TO_EMPTY_DURATION } },
```

Comment:
Before this PR, if a transition contains empty(that's, empty->motion or motion->empty), `getCurrentTransition()` returns null. This PR fixed it.